src/joystick/usb_ids.h

```diff
@@ -46,6 +49,7 @@
 #define USB_VENDOR_SHANWAN      0x2563
 #define USB_VENDOR_SHANWAN_ALT  0x20bc
 #define USB_VENDOR_SONY         0x054c
+#define USB_VENDOR_STMICRO      0x0483
```

Comment:
Or `USB_VENDOR_ST` or `USB_VENDOR_STMICROELECTRONICS`? Is there a policy for how we abbreviate vendors?